### PR TITLE
Notes after IMM installation on k8s deployed via Kubespray

### PR DESCRIPTION
**Pull request by ksankiew**

### CODE REVIEW

File: docs/kubespray_gcp.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Changes required to enable oidc authentication in Kubernetes installed via Kubespray
+
+```
+cd /etc/kubernetes/manifests/
+vi kube-apiserver.yaml
+```
+Add following flags with desired values
+
+[More information about oidc flags here](../blob/master/docs/deployment.md)
```
Comment (pawel-pieniazek) [inline docs/kubespray_gcp.md]: link does not work

File: docs/kubespray_gcp.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## Deploy Kubernetes with Kubespray
+## Example on Google Cloud Platform
```
Comment (pawel-pieniazek) [inline docs/kubespray_gcp.md]: one more `#` here

Comment (pawel-pieniazek) [reply docs/kubespray_gcp.md]: this this not `baremetal` config?
We also talk to add link to - https://github.com/kubernetes/ingress-nginx/blob/master/docs/deploy/baremetal.md or this is not needed?
Maybe another section?

File: docs/kubespray_gcp.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Add following flags with desired values
+
+[More information about oidc flags here](../blob/master/docs/deployment.md)
+
+```
+--oidc-issuer-url
+--oidc-client-id
+--oidc-groups-claim
+--oidc-username-claim
+--oidc-ca-file # path to file
```
Comment (pawel-pieniazek) [inline docs/kubespray_gcp.md]: add comment to all commends or not add at all :) not just one description

File: docs/kubespray_gcp.md (hunk 2 above)
Comment (mareklevv) [reply docs/kubespray_gcp.md]: If you were thinking about nginx used in our Ingress then baremetal is described in [Ingress subchart readme](https://github.com/IntelAI/inference-model-manager/blob/master/helm-deployment/ing-subchart/README.md)

File: docs/kubespray_gcp.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+```
+After saving changes in kube-apiserver.yaml file, kubeapi server pod will be restarted automatically, what means that until pod is up & running again, kubectl commands will return connection refused errors.
+```
+kubectl get pods -n kube-system
+The connection to the server <IP_ADDRESS>:6443 was refused - did you specify the right host or port?
+```
+If error occurs after few minutes, review updates in kube apiserver configuration and double check path to certificate file. Remember, to not to add `oidc-ca-file` flag before placing certificate file in right directory as kube apiserver pod will not start.
```
Comment (mareklevv) [inline docs/kubespray_gcp.md]: Maybe it is worth adding a separate troubleshooting section here?

**Pull request merged**